checker/jdk/Makefile.jdk

```diff
@@ -62,7 +62,7 @@ validate: classes $(DIFFS)
 # does not exist.  So long as the variables are not used on such a make
 # invocation, this doesn't cause a problem, but it is irritating (and a bit
 # sloppy).
-BUILD_CLASSES = $(shell find build/java build/javax -name '*.class')
+BUILD_CLASSES = $(shell find build/java build/javax build/org/w3c build/org/xml -name '*.class')
```

Comment:
Can we have a single variable that contains the packages "java javax com org" and then use Makefile magic to either prepend "src" or "build" to each one? That would reduce the duplication in these two locations.